### PR TITLE
OPENEUROPA-1521: Use PHP CS Fixer.

### LINKED ISSUE #49: Use PHP CS Fixer
Configuration helper: https://mlocati.github.io/php-cs-fixer-configurator/

Get a PR diff without the whitespace changes: https://github.com/openeuropa/code-review/pull/50/files?utf8=%E2%9C%93&diff=split&w=1

### PR DESCRIPTION
**Pull request by drupol**
Fix #49

### CODE REVIEW

File: dist/drupal-conventions.yml [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-    - module
-    - inc
-    - theme
-  tasks.phpmd.triggered_by:
-    - php
-    - module
-    - inc
-    - theme
+    tasks.phpcs.standard: ./vendor/drupal/coder/coder_sniffer/Drupal
+    tasks.phpcs.triggered_by:
```
Comment (voidtek) [inline dist/drupal-conventions.yml]: We should add these 3 following extensions as `phpcsfixer`
-`install`
-`module`
-`profile`

File: resources/php_cs_fixer.library.dist [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
```
Comment (voidtek) [inline resources/php_cs_fixer.library.dist]: This configuration seems adapted for `drupal` and not for `library`

File: dist/drupal-conventions.yml (hunk 1 above)
Comment (drupol) [reply dist/drupal-conventions.yml]: Fixed.

File: resources/php_cs_fixer.library.dist (hunk 2 above)
Comment (drupol) [reply resources/php_cs_fixer.library.dist]: Fixed.

Comment (drupol) [reply resources/php_cs_fixer.library.dist]: Indeed, it's fixed now.

File: tests/AbstractTest.php [hunk 3]
```diff
@@ -16,14 +18,14 @@ abstract class AbstractTest extends TestCase
      * Get fixture file object.
      *
      * @param string $fixture
-     *   Fixture file name.
+     *   Fixture file name
```
[the remark this answers is not in the document]
Comment (drupol) [reply tests/AbstractTest.php]: It's used by the **Symfony code standard**.
See the directive: `phpdoc_annotation_without_dot` on the page: https://github.com/FriendsOfPHP/PHP-CS-Fixer

**Comment by drupol** [comment]
@voidtek I think that the timeout you got was because we didn't removed the `web` directory from the list of directories to scan, which has been done in the latest commits (see https://github.com/openeuropa/code-review/pull/50/commits/bc5f4ed92364c27c24dd92e45bf2937aa263e88f).
However, I will push a commit and increase the default timeout (60 sec).

File: tests/AbstractTest.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (drupol) [reply tests/AbstractTest.php]: I disabled the directive as discussed lively.

**Comment by voidtek** [comment]
Before to merge it, please create a second branch "1.x" - to have the support of this version.
After you can create a major release "2.0.0".

**Pull request merged**